libenv/unix_iface.c

```diff
@@ -752,4 +755,370 @@ static int aix_get_mac_addr(const char *device_name, uint8_t mac[6])
 }
 #endif /* _AIX */
 
+/*******************************************************************/
+
+// returns NULL on any failure
+JsonElement* StringCaptureData(const char* regex, const char* data)
```

Comment:
Should be `static`. This goes for all functions that are not exported via the `unix_iface.h` file.